content/en/blog/multiplexer/index.md

```diff
@@ -0,0 +1,326 @@
+---
+date: 2022-09-30
+title: "Ledger Multiplexer"
+linkTitle: "Ledger Multiplexer"
+description: "I implemented a ledger multiplexer in Go for Indy SDK (libindy).
```

Comment:
This is a bit hard understand when first there is first-person-form and then it changes like imperative "Be able"

Reply:
agree, but trusted Grammarly :-D Have to read I the 100th time and just make a decision.